integration/test/helper.js

```diff
@@ -175,17 +165,18 @@ beforeAll(async () => {
 
 afterEach(async () => {
   await Parse.User.logOut();
-  // Connection close events are not immediate on node 10+... wait a bit
-  await sleep(0);
-  if (Object.keys(openConnections).length > 0) {
-    console.warn('There were open connections to the server left after the test finished');
-  }
   Parse.Storage._clear();
   await TestUtils.destroyAllDataPermanently(true);
-  destroyAliveConnections();
```

Comment:
Instead of removing it, would it make sense to call it in `afterAll` if there are any open connections? After testing, why would we want to keep any open Jasmine connection?

Reply:
After testing Jasmine process ends, the server shutsdown, and there is nothing to destroy.

Reply:
Got it